### DISCUSSION TITLE
Configuring multiple master nodes

### DISCUSSION THREAD

**Discussion by GustWind-byte** [jhuckaby/Cronicle#831] [Q&A] [answered]
Configuring multiple master nodes
Can I use redis?
If so, how to access it?

**Comment by jhuckaby** [comment, marked as answer]
Alas, I'm so sorry, but it's not officially supported.

But if you want to live a little dangerously, check out:

https://github.com/jhuckaby/pixl-server-storage#redis

This is the storage system that powers Cronicle under the hood, and it does technically support Redis.  Just please note that Cronicle was never tested with a Redis storage backend, so there may be unforeseen issues.

If you want to try it, set up your Cronicle `Storage` configuration like this:

```json
"Storage": {
	"engine": "Redis",
	"Redis": {
		"host": "127.0.0.1",
		"port": 6379,
		"keyPrefix": "cronicle",
		"keyTemplate": ""
	}
}
```

But you'll need to follow the other instructions in the linked docs, especially installing the `ioredis` NPM module, etc.

Once you have all that configured, run the setup to initialize Redis: https://github.com/jhuckaby/Cronicle/blob/master/docs/Setup.md#setup

**Reply by GustWind-byte** [reply]
OK, I roughly understand. I currently deploy a single master and multiple slaves. Now the master is down and the whole system is down. I am very upset.

**Comment by jhuckaby** [comment]
We only officially support NFS (shared Filesystem), Amazon S3, or Couchbase, for multi-master setups.  Please use Redis at your own risk.

**Reply by GustWind-byte** [reply]
Thank you for your answer. I will not use redis anymore. I will study NFS. Thank you.
